src/Microsoft.DotNet.Darc/src/Darc/Constants.cs

```diff
@@ -51,35 +46,35 @@ public class Constants
             "Each policy may have a set of required properties." +
             "See below for available merge policies:",
             "",
-            $"{StandardMergePolicyName} - Corresponds to either StandardGitHub or StandardAzureDevOps depending on the target repo type",
+            $"{MergePolicyConstants.StandardMergePolicyName} - Corresponds to either StandardGitHub or StandardAzureDevOps depending on the target repo type",
```

Comment:
I really appreciate these constants getting broken up into more reasonably encapsulated types. :-)  No change... just wanted to say I was happy. :-)